rpmlint/checks/PolkitCheck.py

```diff
@@ -0,0 +1,128 @@
+import os
+from re import split
+from xml.dom.minidom import parse
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class PolkitCheck(AbstractCheck):
+    def __init__(self, config, output):
+        super().__init__(config, output)
+        self.polkit_privs_whitelist = config.configuration.get('PolkitPrivsWhitelist', ())
```

Comment:
it is needed for `check_perm_files` because it contains whitelisted paths from `/etc/polkit-default-privs.d/` . if we remove the part with `/etc/polkit-default-privs.d/`, this could be removed too.

The whitelisting variable was named `POLKIT_RULES_WHITELIST` and is already removed with all the filedigests stuff.

Reply:
I have removed the perm file feature in the new PR #649